src/enclave-agent/src/main.rs

```diff
@@ -56,14 +59,47 @@ impl ImageService {
             validate::verify_id(&cid)?;
         }
 
+        let keyprovider_config = Path::new("/etc").join("ocicrypt_keyprovider_native.conf");
+        if !keyprovider_config.exists() {
+            let config = r#"
+            {
+                "key-providers": {
+                    "attestation-agent": {
+                        "native": "attestation-agent"
+                    }
```

Comment:
Can the config be something like:
            {
                "key-providers": {
                    "attestation-agent": {
                        "native": "<kbc_name>:<kbs_uri>"
                    }
                }
            }
For example, in the eaa_kbc mode of attestation agent, it should be something like "eaa_kbc:127.0.0.1:1234"

Reply:
I think that KBC instance is not configed here while configed like this `provider:attestation-agent:eaa_kbc::127.0.0.1:1234` in decrypt_config.json